File: spec/code_samples/Python python3/keys/post.py
import http.client

conn = http.client.HTTPConnection("ec2-3-222-208-106.compute-1.amazonaws.com:8086")

payload = "{\"exchange\":\"kraken\",\"key\":\"1aa5695d-4b8d-4aeb-8312-8a97d2119764\",\"secret\":\"1aa5695d-4b8d-4aeb-8312-8a97d2119764\",\"passphrase\":\"SECRET PHRASE\",\"activated\":true,\"allocation\":\"TIER4\"}"

headers = { 'content-type': "application/json" }

conn.request("POST", "/keys", payload, headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))

File: spec/code_samples/Node request/exchange-keys/post.js
var request = require("request");

var options = {
  method: 'POST',
  url: 'http://ec2-3-222-208-106.compute-1.amazonaws.com:8086/exchange-keys',
  headers: {'content-type': 'application/json'},
  body: {
    exchange: 'kraken',
    key: '1aa5695d-4b8d-4aeb-8312-8a97d2119764',
    secret: '1aa5695d-4b8d-4aeb-8312-8a97d2119764',
    passphrase: 'SECRET PHRASE',
    activated: true,
    allocation: 'TIER4'
  },
  json: true
};

request(options, function (error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});
